spec/services/preserved_object_handler_create_spec.rb

```diff
@@ -70,7 +70,7 @@
             expect(results).to include(a_hash_including(result_code => a_string_matching('foo')))
           end
           it 'does NOT get CREATED_NEW_OBJECT message' do
-            expect(results).to include(hash_not_including(PreservedObjectHandler::CREATED_NEW_OBJECT))
+            expect(results).not_to include(hash_including(PreservedObjectHandler::CREATED_NEW_OBJECT))
```

Comment:
this is a slightly interesting twist on how best to express that we don't expect something in the results.  I think I prefer your way.

Reply:
well, the thing is, the app code changes above are a result of this change:  this change without those caused test failures, because i think this is expressing a slightly different expectation, which is unmet by the previous version of the app code.

i agree that the app code changes above are less readable, but i think they're more correct.

would you agree that this test change is correct?  and that it expresses something different than the prior version (expecting not to include a specific code, instead of expecting to include a code which is not the code in question, even if the code in question is also in the list).

this test change feels like the crux of this commit to me, and the other things feel like necessary follow ons.